init.lua

```diff
@@ -119,7 +119,9 @@ local Direction <const> = {
     UP = -2,
     DOWN = 2,
     WIDTH = 3,
-    HEIGHT = 4
+    HEIGHT = 4,
+    ASCENDING = "ascending",
```

Comment:
Oh, could you assign these an integer value, eg. ASCENDING = 5 and DESCENDING = 6. Their names make the purpose clear, so there's no need for a string value.